db/schema.rb

```diff
@@ -55,6 +55,33 @@
   add_index "identities", ["session_uuid"], name: "index_identities_on_session_uuid", unique: true, using: :btree
   add_index "identities", ["user_id"], name: "index_identities_on_user_id", using: :btree
 
+  create_table "piis", force: :cascade do |t|
```

Comment:
What about "attributes".  I know its generic.. but I kind like it.  I can imagine thinks "what are the verified attributes of this user" when i write my User.attributes(where="verified=true").